Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
Hmm yeah, but instead of extending BaseVM, why not make it an independent protocol?
CountDownTimer will take in a VM object conforming to that independent protocol.